docs/source/gateway/index.rst

```diff
@@ -1,6 +1,782 @@
+| **openai_api_base**        | Yes      |         | This is the base URL for the Azure OpenAI API service provided by Azure.                      |
++----------------------------+----------+---------+-----------------------------------------------------------------------------------------------+
+| **openai_api_version**     | Yes      |         | The version of the Azure OpenAI service to utilize, specified by a date.                      |
++----------------------------+----------+---------+-----------------------------------------------------------------------------------------------+
+| **openai_deployment_name** | Yes      |         | This is the name of the deployment resource for the Azure OpenAI service.                     |
++----------------------------+----------+---------+-----------------------------------------------------------------------------------------------+
+| **openai_organization**    | No       |         | This is an optional field to specify the organization in OpenAI.                              |
++----------------------------+----------+---------+-----------------------------------------------------------------------------------------------+
+
+
+An example configuration for Azure OpenAI is:
+
+.. code-block:: yaml
+
+    routes:
+      - name: completions
+        route_type: llm/v1/completions
+        model:
+          provider: openai
+          name: gpt-35-turbo
+          config:
+            openai_api_type: "azuread"
+            openai_api_key: $AZURE_AAD_TOKEN
+            openai_deployment_name: "{your_deployment_name}"
+            openai_api_base: "https://{your_resource_name}-azureopenai.openai.azure.com/"
+            openai_api_version: "2023-05-15"
+
+
+.. note::
+
+    Azure OpenAI has disctinct features as compared with the direct OpenAI service. For an overview, please see `the comparison documentation <https://learn.microsoft.com/en-gb/azure/cognitive-services/openai/how-to/switching-endpoints>`_.
+
+For specifying an API key, there are three options:
+
+1. (Preferred) Use an environment variable to store the API key and reference it in the YAML configuration file. This is denoted by a ``$`` symbol before the name of the environment variable.
+2. (Preferred) Define the API key in a file and reference the location of that key-bearing file within the YAML configuration file.
+3. Directly include it in the YAML configuration file.
+
+.. important::
+
+    The use of environment variables or file-based keys is recommended for better security practices. If the API key is directly included in the configuration file, it should be ensured that the file is securely stored and appropriately access controlled.
+    Please ensure that the configuration file is stored in a secure location as it contains sensitive API keys.
+
+Querying the AI Gateway
+=======================
+
+Once the MLflow AI Gateway server has been configured and started, it is ready to receive traffic from users.
```

Comment:
I think this is where we should document the standard gateway parameters and ranges from https://docs.google.com/document/d/1AEDlqSWh4n0j1Ue-llk03aRz7-xBK4P4Zh-Yvo1uhnI/edit#heading=h.5xlyhqfr1sqd and then also indicate that provider-specific parameters can also be specified & include an example